src/olympia/devhub/tests/test_views.py

```diff
@@ -230,12 +230,12 @@ def test_no_sort_updated_filter_for_themes(self):
         addon = addon_factory(type=amo.ADDON_PERSONA)
         addon.addonuser_set.create(user=self.user_profile)
 
-        # There's no "updated" sort filter, so order by the default: "Name".
-        response = self.client.get(self.themes_url + '?sort=updated')
+        # There's no "updated" sort filter, so order by the default: "Created".
+        response = self.client.get(self.themes_url)
```

Comment:
You don't need to update this test - it's checking that a request for `sort=updated` becomes the default sort (created now, name before)